biojava-structure/src/main/java/org/biojava/nbio/structure/chem/DownloadChemCompProvider.java

```diff
@@ -320,8 +319,13 @@ private static boolean downloadChemCompRecord(String recordName) {
             try (PrintWriter pw = new PrintWriter(new GZIPOutputStream(new FileOutputStream(newFile)));
                  BufferedReader fileBuffer = new BufferedReader(new InputStreamReader(uconn.getInputStream()))) {
                 String line;
+                boolean success = false;
                 while ((line = fileBuffer.readLine()) != null) {
                     pw.println(line);
+                    success = true;
+                }
+                if(!success) {
+                	throw new IOException("Could not read from URL "+url.toString());
```

Comment:
Could you be more specific in the error message: `"Found no content in URL" + ...`

Reply:
OK. I will mention the two possible reasons